pkgs/os-specific/linux/vmware/default.nix

```
@@ -0,0 +1,49 @@
+{ lib, stdenv, fetchFromGitHub, kernel, kmod, gnugrep }:
+
+stdenv.mkDerivation rec {
+  pname = "vmware-modules";
+  version = "16.2.3-${kernel.version}";
+
+  src = fetchFromGitHub {
+    owner = "mkubecek";
+    repo = "vmware-host-modules";
+    rev = "4af1a71978962f9805fe2e7e6ceb05c24f42c7f0";
+    sha256 = "sha256-EM6YU2nOwNlAXpQ7cGrLS1N+gAS1KxleVjJTzo22De0=";
+  };
+
+  hardeningDisable = [ "pic" ];
+
+  nativeBuildInputs = kernel.moduleBuildDependencies;
+
+  enableParallelBuilding = true;
+
+  prePatch = ''
+
+    substituteInPlace "./Makefile" \
```

```suggestion
  postPatch = ''
    substituteInPlace Makefile \
```
We don't need to quote or use relative path here.

---

quotes and relative path removed